kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/resources/KubernetesResourceResolver.java

```diff
@@ -0,0 +1,35 @@
+package org.arquillian.cube.kubernetes.impl.resources;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+public class KubernetesResourceResolver {
```

Comment:
Now let's submit as is, but can you open a new issue to reduce the technical debt of this and create a common class that can be used everywhere? `Istio`, `Templates`, `OpenShiftResources` and `KubernetesResources`?

Reply:
Handled as part of this PR itself.